evolvedbinary/prosemirror-lwdita
# Introduce the authentication process to the octokit module

This PR introduces the mechanism to exchange user code with an auth token.

# how to test this PR
* checkout the branch `octokit/auth`
* install any new dependencies by running `yarn install`
* add environment variables to your code ( DM me to get them).
* Make a request to `https://github.com/apps/petal-demo/installations/new`
* copy the given code by GitHub
* make a request to `http://localhost:3000/api/github/token?code=`
* the site should return an auth token as a response.


---
This PR is part of the octokit module.
